envi/__init__.py

```diff
@@ -1169,6 +1176,7 @@ def getArchModules(default=ARCH_DEFAULT):
     import envi.archs.amd64 as e_amd64
     import envi.archs.thumb16 as e_thumb16
     import envi.archs.msp430 as e_msp430
+    import envi.archs.h8 as e_h8
```

Comment:
woops?  how did that escape me!  maybe i was shy about pre-empting all the other architectures ;)